src/features/users/user-route.ts

```diff
@@ -15,8 +15,10 @@ import AlreadyExistsError from "../../exceptions/already-exists-error";
 router.post('/login',
     validate(loginValidation),
     passport.authenticate('local'),
-    asyncHandler(async (req: any, res: Response, next: NextFunction) => {
-        const newSession = req.session.passport.user;
+    asyncHandler(async (req: Request, res: Response, next: NextFunction) => {
+        // TODO fix Request object for session
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Do you still need this disable-line?

Reply:
Looks like it from here, but not from the source, maybe this is stale